utils/command_build_clib.py

```diff
@@ -270,8 +279,10 @@ def build_libraries(self, libraries):
                     link_command += " /link"  # start linker options
                     link_command += " " + " ".join(extra_link_preargs)
                     link_command += " " + ".lib ".join(libraries) + ".lib"  # libraries
+                    link_command += " /LIBPATH:" + " /LIBPATH:".join(library_dirs)
                     link_command += " /OUT:" + c_library_filename  # output file name
                     link_command += " " + " ".join(extra_link_postargs)
+                    print(link_command)
```

Comment:
We don't have to keep prints in the code.

Reply:
This print helps to understand what happens in the log because command itself is not printed during execution with `os.system`.